akka-docs/rst/java/http/client-side/websocket-support.rst

```diff
@@ -83,4 +87,31 @@ underlying TCP interface. The same scenarios as described for regular HTTP reque
 
 The returned layer forms a ``BidiFlow<Message, SslTlsOutbound, SslTlsInbound, Message, CompletionStage<WebSocketUpgradeResponse>>``.
 
+.. _half-closed-client-websockets-java:
+
+
+Half-Closed WebSockets
+----------------------
+The WebSocket API does not support half-closed connections, which means that if the either stream completes, the
+entire connection is closed (after a "Closing Handshake" has been exchanged or a timeout of 3 seconds has passed).
```

Comment:
It isn't an idle timeout, it is send close-handshake, wait for close-handshake back or timeout (which actually misses the first 3 second timeout and always lands at 6 seconds).